### PR TITLE
View

### PR DESCRIPTION
**Pull request by NoFaceBoy**

### CODE REVIEW

File: view/__init__.py
Comment (IgorBrat) [inline view/__init__.py:1]: If you use only <model>_blueprint from views, it's better to import only it: not to expose other methods from the module
Smthng like ```from .view import <model>_view.<model>_blueprint```

File: view/car_view.py
Comment (IgorBrat) [inline view/car_view.py:1]: I don't know if it's possible to inject views with **Repository** and **Controller** objects. It's just a good practice to create these objects on app level (in *main.py* or *app.py*), make them *Singleton*, and from there parse them to views. If it's not possible, then your implementation is OK.

**Review by IgorBrat — COMMENTED** [review]
Please look at comments, and contact me about parsing objects to views as I described in one comment.

File: view/car_view.py
Comment (IgorBrat) [reply view/car_view.py:1]: Also for now don't make them Singleton, we'll refactor it later.

File: view/__init__.py
Comment (NoFaceBoy) [reply view/__init__.py:1]: Done.

File: view/car_view.py
Comment (NoFaceBoy) [reply view/car_view.py:1]: Done.

**Comment by peopleAlreadyKnowWhoIAm** [comment]
Plz add api prefix for all endpoints (for example `/api/accounts`) to differ them from page routes

**Pull request merged**